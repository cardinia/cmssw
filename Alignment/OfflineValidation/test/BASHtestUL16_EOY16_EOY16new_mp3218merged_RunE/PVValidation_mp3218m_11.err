26-Sep-2019 12:56:29 CEST  Initiating request to open file root://eoscms.cern.ch//eos/cms/store/data/Run2016E/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/10000/F0EE6221-5A07-E911-B26C-002590D9D9FC.root
26-Sep-2019 12:56:35 CEST  Successfully opened file root://eoscms.cern.ch//eos/cms/store/data/Run2016E/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/10000/F0EE6221-5A07-E911-B26C-002590D9D9FC.root
Begin processing the 1st record. Run 276945, Event 199850, LumiSection 1 on stream 0 at 26-Sep-2019 12:56:55.213 CEST
Begin processing the 1001st record. Run 276946, Event 6790480, LumiSection 6 on stream 0 at 26-Sep-2019 12:58:26.623 CEST
Begin processing the 2001st record. Run 276946, Event 14695082, LumiSection 13 on stream 0 at 26-Sep-2019 12:58:30.403 CEST
Begin processing the 3001st record. Run 276946, Event 22852929, LumiSection 19 on stream 0 at 26-Sep-2019 12:58:34.615 CEST
Begin processing the 4001st record. Run 276946, Event 31212253, LumiSection 25 on stream 0 at 26-Sep-2019 12:58:37.701 CEST
Begin processing the 5001st record. Run 277071, Event 274709425, LumiSection 147 on stream 0 at 26-Sep-2019 12:58:40.420 CEST
Begin processing the 6001st record. Run 277071, Event 284542412, LumiSection 152 on stream 0 at 26-Sep-2019 12:58:43.270 CEST
Begin processing the 7001st record. Run 277071, Event 290934659, LumiSection 156 on stream 0 at 26-Sep-2019 12:58:46.356 CEST
Begin processing the 8001st record. Run 277071, Event 298149844, LumiSection 160 on stream 0 at 26-Sep-2019 12:58:48.831 CEST
Begin processing the 9001st record. Run 277071, Event 307085187, LumiSection 166 on stream 0 at 26-Sep-2019 12:58:51.791 CEST
Begin processing the 10001st record. Run 277072, Event 813431829, LumiSection 460 on stream 0 at 26-Sep-2019 12:58:54.254 CEST
Begin processing the 11001st record. Run 277072, Event 820550620, LumiSection 464 on stream 0 at 26-Sep-2019 12:58:56.413 CEST
Begin processing the 12001st record. Run 277076, Event 1631219915, LumiSection 1023 on stream 0 at 26-Sep-2019 12:58:58.788 CEST
Begin processing the 13001st record. Run 277076, Event 1642320140, LumiSection 1031 on stream 0 at 26-Sep-2019 12:59:00.563 CEST
Begin processing the 14001st record. Run 277076, Event 1649952771, LumiSection 1036 on stream 0 at 26-Sep-2019 12:59:01.978 CEST
Begin processing the 15001st record. Run 277087, Event 1229433851, LumiSection 1139 on stream 0 at 26-Sep-2019 12:59:04.103 CEST
Begin processing the 16001st record. Run 277087, Event 1234390091, LumiSection 1143 on stream 0 at 26-Sep-2019 12:59:06.539 CEST
Begin processing the 17001st record. Run 277087, Event 1239460994, LumiSection 1148 on stream 0 at 26-Sep-2019 12:59:09.060 CEST
Begin processing the 18001st record. Run 277087, Event 1253297483, LumiSection 1160 on stream 0 at 26-Sep-2019 12:59:11.631 CEST
Begin processing the 19001st record. Run 277087, Event 1256970789, LumiSection 1164 on stream 0 at 26-Sep-2019 12:59:13.628 CEST
Begin processing the 20001st record. Run 277087, Event 1262779536, LumiSection 1169 on stream 0 at 26-Sep-2019 12:59:15.017 CEST
Begin processing the 21001st record. Run 277087, Event 1266577743, LumiSection 1173 on stream 0 at 26-Sep-2019 12:59:16.567 CEST
Begin processing the 22001st record. Run 277087, Event 1271647312, LumiSection 1177 on stream 0 at 26-Sep-2019 12:59:18.374 CEST
Begin processing the 23001st record. Run 277087, Event 1276877886, LumiSection 1182 on stream 0 at 26-Sep-2019 12:59:20.287 CEST
Begin processing the 24001st record. Run 277087, Event 1280961755, LumiSection 1186 on stream 0 at 26-Sep-2019 12:59:21.724 CEST
Begin processing the 25001st record. Run 277087, Event 1285533116, LumiSection 1190 on stream 0 at 26-Sep-2019 12:59:23.428 CEST
Begin processing the 26001st record. Run 277096, Event 2176297137, LumiSection 1247 on stream 0 at 26-Sep-2019 12:59:26.350 CEST
Begin processing the 27001st record. Run 277096, Event 2184084824, LumiSection 1252 on stream 0 at 26-Sep-2019 12:59:28.308 CEST
Begin processing the 28001st record. Run 277096, Event 2190846084, LumiSection 1257 on stream 0 at 26-Sep-2019 12:59:30.755 CEST
Begin processing the 29001st record. Run 277096, Event 2201278792, LumiSection 1263 on stream 0 at 26-Sep-2019 12:59:34.390 CEST
Begin processing the 30001st record. Run 277126, Event 17326132, LumiSection 44 on stream 0 at 26-Sep-2019 12:59:36.842 CEST
Begin processing the 31001st record. Run 277126, Event 25761783, LumiSection 53 on stream 0 at 26-Sep-2019 12:59:40.243 CEST
Begin processing the 32001st record. Run 277202, Event 792022293, LumiSection 505 on stream 0 at 26-Sep-2019 12:59:42.733 CEST
26-Sep-2019 12:59:44 CEST  Closed file root://eoscms.cern.ch//eos/cms/store/data/Run2016E/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/10000/F0EE6221-5A07-E911-B26C-002590D9D9FC.root

=============================================

MessageLogger Summary

 type     category        sev    module        subroutine        count    total
 ---- -------------------- -- ---------------- ----------------  -----    -----
    1 fileAction           -s file_close                             1        1
    2 fileAction           -s file_open                              2        2

 type    category    Examples: run/evt        run/evt          run/evt
 ---- -------------------- ---------------- ---------------- ----------------
    1 fileAction           PostGlobalEndRun                  
    2 fileAction           pre-events       pre-events       

Severity    # Occurrences   Total Occurrences
--------    -------------   -----------------
System                  3                   3

dropped waiting message count 0
[6.887MB/6.887MB][100%][==================================================][6.887MB/s]  [6.887MB/6.887MB][100%][==================================================][6.887MB/s]  
