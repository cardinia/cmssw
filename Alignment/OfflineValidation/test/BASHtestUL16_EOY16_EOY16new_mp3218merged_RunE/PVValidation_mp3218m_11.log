000 (3189561.011.000) 09/26 11:23:32 Job submitted from host: <137.138.44.75:9618?addrs=137.138.44.75-9618+[2001-1458-d00-1a--b]-9618&alias=bigbird14.cern.ch&noUDP&sock=schedd_2873_37ae_40>
...
040 (3189561.011.000) 09/26 12:55:45 Started transferring input files
	Transferring to host: <188.184.139.89:9618?addrs=188.184.139.89-9618+[2001-1458-301-45--100-155]-9618&noUDP&sock=7659_3551_76267>
...
040 (3189561.011.000) 09/26 12:55:45 Finished transferring input files
...
001 (3189561.011.000) 09/26 12:55:45 Job executing on host: <188.184.139.89:9618?addrs=188.184.139.89-9618+[2001-1458-301-45--100-155]-9618&noUDP&sock=5722_6500_3>
...
006 (3189561.011.000) 09/26 12:55:53 Image size of job updated: 22564
	12  -  MemoryUsage of job (MB)
	11748  -  ResidentSetSize of job (KB)
...
040 (3189561.011.000) 09/26 13:00:29 Started transferring output files
...
040 (3189561.011.000) 09/26 13:00:29 Finished transferring output files
...
005 (3189561.011.000) 09/26 13:00:29 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:02:17, Sys 0 00:00:05  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:02:17, Sys 0 00:00:05  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	6861  -  Run Bytes Sent By Job
	1015  -  Run Bytes Received By Job
	6861  -  Total Bytes Sent By Job
	1015  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :                 1           
	   Disk (KB)            :     7097        1   2318634 
	   Memory (MB)          :       12     2000      2000 
...
