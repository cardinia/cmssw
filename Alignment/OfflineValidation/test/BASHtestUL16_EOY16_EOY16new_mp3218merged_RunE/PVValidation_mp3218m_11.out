Job started at  Thu Sep 26 12:55:46 CEST 2019
batch dir: /pool/condor/dir_8531 release: /afs/cern.ch/cms/CAF/CMSALCA/ALCA_TRACKERALIGN/data/commonValidation/alignmentObjects/paconnor/CMSSW_10_6_0/src release base: /cvmfs/cms.cern.ch/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_0
cmsRun PVValidation_mp3218m_276945_cfg.py
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: This is DATA!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Applying TrackerSurfaceDeformations!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Running DA Algorithm!
Content of working dir is total 7.0M -rw-r--r--. 1 tkello zh 6.9M Sep 26 13:00 PVValidation_mp3218m_276945.root -rw-r--r--. 1 tkello zh 20K Sep 26 12:55 PVValidation_mp3218m_276945_cfg.py -rw-r--r--. 1 tkello zh 5.5K Sep 26 13:00 _condor_stderr -rw-r--r--. 1 tkello zh 521 Sep 26 12:56 _condor_stdout -rwxr-xr-x. 1 tkello zh 1015 Sep 26 12:55 condor_exec.exe -rw-------. 1 tkello zh 4.3K Sep 26 12:56 tkello.cc drwx------. 3 tkello zh 4.0K Sep 26 12:55 tmp drwx------. 3 tkello zh 4.0K Sep 26 12:55 var
Job ended at  Thu Sep 26 13:00:29 CEST 2019
